docs/topics/windows-provisioning-scripts-release-notes.md

```diff
@@ -0,0 +1,17 @@
+## v0.0.10
```

Comment:
great Idea to document this!

Reply:
@AbelHu suggested this!
If anyone has a better idea where to store these let me know.